syntax = "proto2";

package MMPro.micromsg;

import "BaseResponse.proto";
import "SKBuiltinString_t.proto";
import "SKBuiltinBuffer_t.proto";
import "SnsUserInfo.proto";
import "CustomizedInfo.proto";
import "SearchContactItem.proto";

message SearchContactResponse {
	required BaseResponse _BaseResponse = 1;
	required SKBuiltinString_t _UserName = 2;
	required SKBuiltinString_t _NickName = 3;
	required SKBuiltinString_t _PYInitial = 4;
	required SKBuiltinString_t _QuanPin = 5;
	required int32 _Sex = 6;
	required SKBuiltinBuffer_t _ImgBuf = 7;
	optional string _Province = 8;
	optional string _City = 9;
	optional string _Signature = 10;
	optional uint32 _PersonalCard = 11;
	optional uint32 _VerifyFlag = 12;
	optional string _VerifyInfo = 13;
	optional string _Weibo = 14;
	optional string _Alias = 15;
	optional string _WeiboNickname = 16;
	optional uint32 _WeiboFlag = 17;
	optional int32 _AlbumStyle = 18;
	optional int32 _AlbumFlag = 19;
	optional string _AlbumBGImgID = 20;
	optional SnsUserInfo _SnsUserInfo = 21;
	optional string _Country = 22;
	optional string _MyBrandList = 23;
	optional CustomizedInfo _CustomizedInfo = 24;
	optional uint32 _ContactCount = 25;
	repeated SearchContactItem _ContactList = 26;
	optional string _BigHeadImgUrl = 27;
	optional string _SmallHeadImgUrl = 28;
	optional SKBuiltinBuffer_t _ResBuf = 29;
	optional string _AntispamTicket = 30;
	optional string _KFWorkerID = 31;
	optional uint32 _MatchType = 32;
}
